src/content/general/architecture/openstack/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 linkTitle: OpenStack
```

Comment:
If the document was made generic, I am guessing the link will require a more generic name too.